### ISSUE TITLE
Configurable Authorization

### ISSUE THREAD

**Issue by Metawolve** [eurofurence/ef-app_backend-dotnet-core#67]
Currently, the authorization is role-based and the roles are written as plain text above each controller method.

We would like to introduce configurable authorization.

I suggest to implement this by using authorization policies with .RequireRole and read the roles from the appsettings.json.
This way we can have the different access rights as policies and can list as many groups as we like in the appsettings.json. If the user is in at least one of these listed groups, he will get access to the resource.

The appsettings.json could look something like this: 

```
"authorization": {
  "developer": [
    "groupA",
    "groupB"
  ],
  "knowledgeBaseMaintainer": [
    "groupA",
    "groupC"
  ],
  ...
}
```

**Comment by Metawolve** [comment]
Was resolved with pull request #90

**Comment by Metawolve** [comment]
Was resolved with pull request #90
